code/datums/components/supermatter_crystal.dm

```diff
@@ -22,6 +24,11 @@
 	src.tool_act_callback = tool_act_callback
 	src.consume_callback = consume_callback
 
+	attackby_hit_typecache = typecacheof(list(
+		/obj/item/melee/roastingstick,
+		/obj/item/toy/crayon/spraycan
+	))
```

Comment:
```suggestion

```

Delete this and move this to the `/datum/component/supermatter_crystal` location.  We shouldn't be creating this list when a SM spawns. (it should be pre-existing)

Reply:
also makes sense